tests/gold_tests/pluginTest/ja3_fingerprint/ja3_fingerprint.test.py

```diff
@@ -134,7 +141,11 @@ def _configure_client(self, tr: 'TestRun') -> None:
         """
         name = f'client{self._client_counter}'
         p = tr.AddVerifierClientProcess(
-            name, self._replay_file, http_ports=[self._ts.Variables.port], https_ports=[self._ts.Variables.ssl_port])
+            name,
+            self._replay_file,
+            http_ports=[self._ts.Variables.port],
+            https_ports=[self._ts.Variables.ssl_port],
+            other_args='--thread-limit 1')
```

Comment:
Why is `--thread-limit 1` needed?

Reply:
I think this will address the log ordering issues we've seen in this test. It makes ATS serialize across the sessions, and this test has an HTTP/1 and HTTP/2 session. Currently there's a race condition as to which finishes first.

Reply:
I broke this change out as a separate PR so that it can get backported to 10.0.x.

https://github.com/apache/trafficserver/pull/11766